foundation/foundation/src/main/res/values-night/styles.xml

```diff
@@ -38,7 +38,16 @@
         <item name="vtmnBorderInformation">@color/vtmn_blue300</item>
         <item name="vtmnBorderPrimaryReversed">@color/vtmn_white</item>
 
-        <item name="vtmnActiveBrand">@color/vtmn_blue600</item>
-        <item name="vtmnActiveAccent">@color/vtmn_yellow100</item>
+        <item name="vtmnActivePrimary">#0382DD</item>
+        <item name="vtmnActivePrimaryTransparent">#263D9ACC</item>
+        <item name="vtmnActiveSecondaryTransparent">#26FFFFFF</item>
+        <item name="vtmnActiveTertiary">#012946</item>
+        <item name="vtmnActiveTertiaryTransparent">#CC012946</item>
+        <item name="vtmnActiveBrand">#266C91</item>
+        <item name="vtmnActiveAccent">#FFF9C7</item>
+        <item name="vtmnActivePrimaryReversedTransparent">#40001018</item>
+        <item name="vtmnActiveSecondaryReversedTransparent">#66001018</item>
+        <item name="vtmnActiveTertiaryReversedTransparent">#99001018</item>
+        <item name="vtmnActiveBrandReversedTransparent">#99001018</item>
```

Comment:
To be sure: You are using hex code because these colors are specific to Android platform and there isn't any semantic color for that?

Reply:
Yes exactly. Comments have been added ✅